contrib/slime/README.org

```diff
@@ -74,6 +74,15 @@ of Common Lisp, you can specify it in your =~/.spacemacs=
 | ~SPC m c f~ | Compile function         |
 | ~SPC m c r~ | Compile region           |
 
+** Definition
+
+| Key Binding               | Description                     |
+|---------------------------+---------------------------------|
+| ~SPC m d d~               | Go to definition                |
```

Comment:
oh... inspect definition ? what does it do exactly ? It may go under `SPC m h i` if it only print some info.